flink/README.md

```diff
@@ -58,6 +58,52 @@ No additional installation is needed on your server.
 
 See [metadata.csv][8] for a list of metrics provided by this integration.
 
+##### Log collection
+
+_Available for Agent >6.0_
+
+1. Flink uses the `log4j` logger by default. To activate logging to a file and customize the format edit the `log4j.properties`, `log4j-cli.properties`, `log4j-yarn-session.properties`, or `log4j-console.properties files`. See [flink's defaults][13] for default configurations. For example, a `log4j.properties` would contain this configuration by default:
+
+    ```
+      log4j.appender.file=org.apache.log4j.FileAppender
+      log4j.appender.file.file=${log.file}
+      log4j.appender.file.append=false
+      log4j.appender.file.layout=org.apache.log4j.PatternLayout
+      log4j.appender.file.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p %-60c %x - %m%n
+    ```
+
+2. By default, our integration pipeline supports the following conversion pattern:
+
+    ```
+      %d{yyyy-MM-dd HH:mm:ss,SSS} %-5p %-60c %x - %m%n
+    ```
+
+    Clone and edit the [integration pipeline][11] if you have a different format.
```

Comment:
Do we usually have customers use their own pipeline? Since we encourage the timestamp in the `log4j.properties` configuration above and it's also flink's default timestamp format, maybe we don't need this?

Reply:
I'm not sure- I included this because I saw it in a different integration that uses log4j and allows users to customize log formats

Reply:
Cool, good to know! Can we also include an example timestamp, I think they might be easier to read than the grok syntax.

Reply:
Sure!